wpilibc/src/main/native/include/frc/LEDPattern.h

```diff
@@ -5,6 +5,7 @@
 #pragma once
 
 #include <functional>
+#include <ranges>
```

Comment:
This can be removed; it was originally added for done experiments with reversed views that didn't wind up being needed